tests/unit/transaction_record_query_test.py

```
@@ -14,6 +14,7 @@
     query_pb2,
     query_header_pb2,
 )
+from hiero_sdk_python.hapi.services import query_header_pb2
```

Ruff (0.14.14)

[error] Redefinition of unused `query_header_pb2` from line 15: `query_header_pb2` redefined here

Remove definition: `query_header_pb2`

(F811)